Added a loading screen feature

ZeyarHtoo opened the pull request:
-Tweaked some code where there is repetitive closing tags 
-Inputted a loading message which will be shown when the submit button is pressed 
- Wrote some javascript for the loading screen and styled the page with css so that there will be a small animation

ZeyarHtoo committed:
Added a loading screen feature

-Tweaked some code where there is repetitive closing tags
-Inputted a loading message which will be shown when the submit button is pressed 
- Wrote some javascript for the loading screen and styled the page with css so that there will be a small animation

kaungkhantko reviewed (approved):
Approved addition to loading screen

Alets17 committed:
Beautify the website

I added style attributes to the website elements to make it more appealing to the user. Also added a small description of our application at the very top of the webpage

ZeyarHtoo committed:
Merge pull request #2 from Alets17/patch-1

Beautify the website
